docs/sources/setup-grafana/configure-security/configure-authentication/azuread.md

```diff
@@ -100,6 +100,30 @@ To enable the Azure AD OAuth2, register your application with Azure AD.
 
 1. Click on **Users and Groups** and add Users/Groups to the Grafana roles by using **Add User**.
 
+### Assign server administrator privileges
+
+> Available in Grafana v9.2 and later versions.
+
+If the application role received by Grafana is `GrafanaAdmin`, the user will be granted server administrator privileges.  
+This is useful if you want to grant server administrator privileges to a subset of users.  
+The user will also be assigned the role of `Admin` of the default organization.
```

Comment:
This is passive voice. Who or what is doing the assigning? In my suggestion, I am making an assumption that the 'actor' is Grafana.